modules/ossm-rn-known-issues.adoc

```diff
@@ -36,6 +37,12 @@ Configuring Federated service meshes for failover does not work as expected. The
 
```

Comment:
We'll also need known issue release notes for:

- https://issues.redhat.com/browse/OSSM-1107 (An issue fixed in 2.1.3, and will be fixed in 2.2.1)
- https://issues.redhat.com/browse/OSSM-1665 (Federation won't work on 4.6 IBM Z/P
- (Possible) https://issues.redhat.com/browse/OSSM-1669

We should confirm the above next week before going out.

Reply:
> Confirmed with leadership team: officially we're just supporting 2.2 on OCP 4.9 and 4.10.

No need to document the Federation KI, since 4.6 isn't supported at all.

Reply:
@SNiemann15, thank you for confirming this. 1665 will not be included in the release notes.

@longmuir, based on comments I've gathered, at this time I'm assuming that 1107 and 1668 are essentially the same thing. Based on other comments, I thought we were including one or the other. 1668 has been added. Do you we think we should also include 1107? If you do, I'm happy to add it.

Reply:
Ack - correct, looks like they're the same. Though I do notice a typo in 1668 above, as `wksResolverCA` should be `jwksResolverCA`,

Reply:
What hack missed that? :)

Thanks, @longmuir , I've made the edit.